ang/civicase/CaseCard--case-list.html

```diff
@@ -18,7 +18,10 @@
       <i ng-if="data.selected" class="civicase__checkbox--checked material-icons">check_box</i>
     </span>
     <div contact-card class="civicase__case-card__contact" contacts="data.client"></div>
-    <div class="civicase__case-card__case-id">Case ID: {{ data.id }}</div>
+    <div class="civicase__case-card__case-id">
+      Case ID: {{ data.id }}
+      <i class="material-icons civicase__case-lock" ng-if="data.lock">lock</i>
```

Comment:
Lock icon should not be inside `civicase__case-card__case-id`. Should be outside.